fr/common.json

```
@@ -0,0 +1,452 @@
+    "route": "{{count}} itinéraire",
+    "route_plural": "{{count}} itinéraires"
+  },
+  "glossary": {
+    "ranking": "Rang",
+    "level": "Niveau",
+    "playerName": "Nom de Joueur",
+    "skill": "Compétence",
+    "tier": "Tier",
+    "mostPick": "Plus Joué",
+    "artisan": "Artisanat",
+    "eternityCutoff": "Cut-off Immortel ",
+    "allCharacter": "Tous",
+    "character": "Personnages",
+    "characterLevel": "Niveau",
+    "winRate": "% Vict.",
+    "pickRate": "% Joué",
+    "halfRate": "% Rang Moy.",
+    "perishRate": "% Mortalité",
+    "averageRank": "Rang Moy.",
+    "averageKill": "Élim. Moy.",
+    "averageKillAssistant": "Élim. + Assist. Moy.",
+    "dpm": "DPM",
+    "playTime": "Temps de Jeu",
+    "playCount": "Parties",
+    "freeCharacters": "Rotation hebdomadaire de personnages",
+    "winRateContribution": "Taux de Contrib. à la Victoire ",
+    "halfRateContribution": "Taux de Contrib. au Classement",
+    "deltaMMRContribution": "Contrib. aux Changements de MMR",
+    "tiers": {
+      "none": "Non Classé",
+      "iron": "Fer",
+      "bronze": "Bronze",
+      "silver": "Argent",
+      "gold": "Or",
+      "platinum": "Platine",
+      "diamond": "Diamant",
+      "demigod": "Titan",
+      "eternity": "Immortel"
+    },
+    "tiersShort": {
+      "none": "NC",
+      "iron": "F",
+      "bronze": "Br",
+      "silver": "Ar",
+      "gold": "Or",
+      "platinum": "Pl",
+      "diamond": "Di",
+      "demigod": "Ti",
+      "eternity": "Im"
+    },
+    "queueType": "Mode de Jeu",
+    "queueTypes": {
+      "all": "Tout",
+      "solo": "Solo",
+      "duo": "Duo",
+      "trio": "Escouade"
+    },
+    "gameType": {
+      "normal": "Normal",
+      "rank": "Classé"
+    },
+    "armorCategory": {
+      "weapon": "Arme",
+      "chest": "Buste",
+      "head": "Tête",
+      "arm": "Bras",
+      "leg": "Jambe",
+      "trinket": "Accessoire"
+    },
+    "itemCategory": {
+      "Weapon": "Armes",
+      "Armor": "Armures",
+      "Consumable": "Consommable",
+      "Special": "Spécial",
+      "Misc": "Divers"
+    },
+    "statCategory": {
+      "Attack": "Attaque",
+      "Skill": "Compétences",
+      "Defense": "Défense",
+      "Misc": "Autres"
+    },
+    "subcategories": {
+      "else": "Autres"
+    }
+  },
+  "global": {
+    "notEnoughSamples": "NC",
+    "close": "Fermer",
+    "retry": "Réessayer",
+    "selectAll": "Sélectionner Tout",
+    "deselectAll": "Déselectionner Tout",
+    "playerSearchPlaceholder": "Chercher un joueur",
+    "error": {
+      "short": "Oups!",
+      "tryAgainLater": "Merci de réessayer plus tard.",
+      "somethingsWrong": "Le site web a rencontré une erreur. Merci de nous contacter si le problème persiste.",
+      "inMaintenance": "Désolé! Maintenance en cours.",
+      "erServerError": "Erreur de communication avec le serveur de l'API Eternal Return."
+    }
+  },
+  "header": {
+    "nav": {
+      "leaderboard/mmr": "Top Joueurs",
```

You wrote `Top` only here, while you wrote `TOP` to the other fields (e.g., TOP 1000).
I'd like to recommend `Classement MMR` since you wrote the title of the page like so.
```suggestion
      "leaderboard/mmr": "Classement MMR",
```

---

It got changed accordingly to your suggestion